submissions/DaniaB24/friends-app/js/app.js

```diff
@@ -0,0 +1,103 @@
+const cardsOfFriends = document.getElementById("all_humans");
+const searchBar = document.getElementById("search");
+const filterMenu = document.getElementById("filter_menu");
+
+let friends;
+let filteredUsers;
+function startApp() {
+  const URL = "https://randomuser.me/api?results=10";
+  fetch(URL)
+    .then(handleErrors)
+    .then((response) => response.json())
+    .then((users) => {
+      console.log("ok");
+      friends = users.results;
+      filteredUsers = users.results;
+      displayUsers(friends);
+      searchFriends(filteredUsers);
+      filterMenu.addEventListener("click", (event) => {
+        filterSetting(event, friends);
+      });
+    })
+    .catch((error) => {
+      alert("Please reload, Authorization failed : " + error.message);
+      console.log("Authorization failed : " + error.message);
+    });
+}
+function handleErrors(response) {
+  if (!response.ok) {
+    throw Error(response.statusText);
+  }
+  return response;
+}
+
+function filterSetting({ target }, friends) {
+  const sortedResults = sortFriends(target, friends);
+  filterByGender(sortedResults ?? []);
+}
+function displayUsers(friends) {
+  const createCards = friends
```

Comment:
```suggestion
  const userCards = friends
```

`createCards` is `создатьКарточки`. It's not a function anyway